api_eucovidcert.yaml

```diff
@@ -0,0 +1,65 @@
+swagger: "2.0"
+info:
+  version: 0.0.1
+  title: EU Covid Certificate service for IO App
+basePath: /eucovidcert/api/v1
+schemes:
+  - https
+security:
+  - Bearer: []
+paths:
+  /certificate:
+    post:
+      operationId: getCertificate
+      summary: Retrieve a certificate for a given Citizen
+      description: |-
+        Given a Citizen's fiscal code and an OTP (the auth code previously sent via IO Message), a Certificate is returned with ID, encoded QR-code, expiration date and a markdown text with all meaningful information to be shown to Citizens.
+      parameters:
+        - name: accessData
+          in: body
+          schema:
+            $ref: "#/definitions/GetCertificateParams"
+      responses:
+        '200':
+          description: |-
+            A Certificate exists and it's found for the given access data. It is retrieved regardless of it's expired or its current status
+          schema:
+            $ref: "#/definitions/Certificate"
+        '400':
+          description: Payload has bad format
+        '403':
+          description: Access data provided are invalid or no Certificate has been emitted for the given Citizen
+        '410':
```

Comment:
What's this for? Let's consider `501 Not Implemented` if we mean to comunicate this functionality is not available (being too early or too late)

Reply:
We agreed with frontend guys to add '410' as status code that defines when this service will be no longer avaiable. 
Sound fine?

Reply:
I'm happy if you are happy

Reply:
Just a note:
`410 Gone` means there was a resource that used to be there, but it's not anymore. Semantically, it refers to the specific resource, hence the specific certificate for the given `auth_code`. `4xx` errors in general tell nothing about the endpoint overall, it's a status regarding the specific request. 

Having that said, if we and the client agreed on that, it's totally fine for me.

Reply:
We can change with a more appropriate, no problem about that at all